introduction.md

```diff
@@ -1,3 +1,14 @@
+* Git
+* Python
+* JS
+
+
+
+
+
+
+
+
 my favourite path is JavaScript
 
 hi this is ma
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?